### ISSUE TITLE
[🐛 Bug]: Selenium Side Runner can't pass user profile to the Microsoft Edge

### ISSUE THREAD

**Issue by rdomingo11** [SeleniumHQ/selenium-ide#1583]
### What happened?

I'm running this command:
selenium-side-runner -c "ms:EdgeOptions.args=[user-data-dir='C:\\Users\randolphd\AppData\Local\Microsoft\Edge\User Data\', profile-directory='Default', use-chromium=true], browserName=MicrosoftEdge" -s "[http://127.0.0.1:9515"](http://127.0.0.1:9515%22/) --debug *.side

With the following versions:
npm = 9.2.0
nvm = 1.1.10
node = v19.4.0
EdgeWebDriver = v108.0.1462.76
Microsoft Edge = v108.0.1462.76

The current policy of the Microsoft Edge is to automatically sign-in using the current windows login account.

However, when I try to run the command above, it is not signing properly AND it is stuck on the user profile page where the user need to select a profile in order to login.

![image](https://user-images.githubusercontent.com/9833439/211426136-bc1489c3-e155-4dab-8993-3a8e95b02e77.png)

This is the what the edge version spawned by the selenium side runner.

![image](https://user-images.githubusercontent.com/9833439/211426246-eb36c2ec-6386-430f-8087-f277cdf48bc7.png)

### How can we reproduce the issue?

```shell
Run this command:
selenium-side-runner -c "ms:EdgeOptions.args=[user-data-dir='C:\\Users\randolphd\AppData\Local\Microsoft\Edge\User Data\', profile-directory='Default', use-chromium=true], browserName=MicrosoftEdge" -s "[http://127.0.0.1:9515"](http://127.0.0.1:9515%22/) --debug *.side
```

### Relevant log output

```shell
Not applicable.
```

### Operating System

Microsoft Windows 10 Enterprise, 10.0.19045

### Selenium version

Selenium Side Runner - 4.0.0-alpha.36

### What are the browser(s) and version(s) where you see this issue?

MicrosoftEdge, v108.0.1462.76

### What are the browser driver(s) and version(s) where you see this issue?

MicrosoftEdge, v108.0.1462.76

### Are you using Selenium Grid?

Using downloadable EdgeWebDriver - version v108.0.1462.76

**Comment by toddtarsi** [comment]
@rdomingo11 - Wow, nice issue description. I'll try and take a look later this week. Thanks for reporting!

**Comment by rdomingo11** [comment]
Any feedback on this ticket? Is this an issue or is it something to do with the configuration?

**Comment by toddtarsi** [comment]
@rdomingo11 - Sorry for the delays here. I am not sure. Could you do the following?

Could you run the side-runner with the debug flags enabled `selenium-side-runner -dD [your-stuff]`?

Basically, if the configuration matches the expectation of what you're supplying, then there might be a selenium-webdriver issue. Otherwise, I have to modify my build argument bootstrapping stuff to handle some case here better

**Comment by rdomingo11** [comment]
This is the configuration. (I'm logging in as administrator)
<img width="837" alt="image" src="https://user-images.githubusercontent.com/9833439/224597223-8b70a072-2e38-41c6-90a9-10aea0a9a2e2.png">

And this is what the Edge Driver is returning. I'm using version 110.0.1587.69 (x64)
<img width="676" alt="image" src="https://user-images.githubusercontent.com/9833439/224597295-0417d58b-6213-4ad6-9dd8-2b4ba6b5c129.png">

**Comment by rdomingo11** [comment]
I'm logging in as a user.
<img width="829" alt="image" src="https://user-images.githubusercontent.com/9833439/224598295-437d0866-8117-46e3-9e7e-c2fd7c7b684b.png">

<img width="829" alt="image" src="https://user-images.githubusercontent.com/9833439/224598432-22e7b369-ed35-48f7-ad34-241182ff2a76.png">

**Comment by rdomingo11** [comment]
Let me know if my current configuration makes sense or not. What is apparent is that it doesn't recognize the user profile that I've passed to the Selenium Side Runner, it assumes that the profile that I'm using is the one I'm currently using.

**Comment by toddtarsi** [comment]
@rdomingo11 - So, one thing I want to make really clear is that I have no idea about any of this stuff. I tend to be in this IDE layer, so the actual parsing of the edge driver stuff isn't in my purview AT ALL. However, I want you to succeed with this stuff, so I want to figure out if my stuff is borked here or not. 

That SE_CONFIGURATION environment variable has a key 'capabilities' that gets passed into selenium-webdriver as the capabilities here:

https://github.com/SeleniumHQ/selenium-ide/blob/80b4e9cb3373f181d4249b091c6da28262f072b5/packages/side-runner/src/main.test.ts#L48

and here:

https://github.com/SeleniumHQ/selenium-ide/blob/80b4e9cb3373f181d4249b091c6da28262f072b5/packages/side-runner/src/run.ts#L46

Basically, if the capabilities key look good on that env var, it's a selenium-webdriver issue.
Otherwise, its a me problem.

**Comment by rdomingo11** [comment]
What's bothering me is the Profile path value is using the Temp rather than the one that I've provided.
![image](https://user-images.githubusercontent.com/9833439/225157826-1e773c65-a627-4e0d-ab72-5c4b123333cd.png)

It looks like it ignored the value that I've passed. Would it be possible to check if value of user-data-dir is passed properly to edgedriver? In the IDE, it works perfectly fine.
